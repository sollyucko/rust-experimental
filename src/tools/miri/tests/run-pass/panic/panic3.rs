// ignore-windows: Unwind panicking does not currently work on Windows
fn main() {
    core::panic!("panicking from libcore");
}
