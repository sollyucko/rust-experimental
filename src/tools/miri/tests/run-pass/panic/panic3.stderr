thread 'main' panicked at 'panicking from libcore', $DIR/panic3.rs:3:5
